lutris/services/steam.py

```diff
@@ -127,29 +127,33 @@ def install_from_steam(self, manifest):
         appid = manifest.steamid
         if appid in self.excluded_appids:
             return
-        service_game = ServiceGameCollection.get_game(self.id, appid)
-        if not service_game:
-            return
-        lutris_game_id = "%s-%s" % (self.id, appid)
-        existing_game = get_game_by_field(lutris_game_id, "installer_slug")
-        if existing_game:
-            return
-        game_config = LutrisConfig().game_level
-        game_config["game"]["appid"] = appid
-        configpath = write_game_config(lutris_game_id, game_config)
-        slug = self.get_installed_slug(service_game)
-        add_game(
-            name=service_game["name"],
-            runner="steam",
-            slug=slug,
-            installed=1,
-            installer_slug=lutris_game_id,
-            configpath=configpath,
-            platform="Linux",
-            service=self.id,
-            service_id=appid,
-        )
-        return slug
+        try:
+            service_game = ServiceGameCollection.get_game(self.id, appid)
+            if not service_game:
+                return
+            lutris_game_id = "%s-%s" % (self.id, appid)
+            existing_game = get_game_by_field(lutris_game_id, "installer_slug")
+            if existing_game:
+                return
+            game_config = LutrisConfig().game_level
+            game_config["game"]["appid"] = appid
+            configpath = write_game_config(lutris_game_id, game_config)
+            slug = self.get_installed_slug(service_game)
+            add_game(
+                name=service_game["name"],
+                runner="steam",
+                slug=slug,
+                installed=1,
+                installer_slug=lutris_game_id,
+                configpath=configpath,
+                platform="Linux",
+                service=self.id,
+                service_id=appid,
+            )
+            return slug
+        except Exception as e:
+            logger.error("Failed to install from Steam: %s", str(e))
```

Comment:
I don't think we need ``str(ex)`` here- just ``ex`` should work. That's what we usually have for ``logger.error``.

Reply:
I'll make the change